### DISCUSSION TITLE
Archiving pl and de locales

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#147] [Announcements]
Hey All,

As you may know, since the move to GitHub the number of languages supported on MDN Web Docs has been reduced. The reason for this was to ensure that the content people _do_ read on MDN was accurate and up to date. We therefore required [certain commitments](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#requirements-for-keeping-locales-up-to-date) from communities who wished to support MDN in their language.

Since then some languages have [gathered a community](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#activating-a-locale) and have had their language added back to MDN. For the Polish and German languages, we've noticed that the documentation efforts have stalled and have now reached a stage that it would not make sense to keep serving the documentation in its current state. What we mean by that is that:

1. The documentation is so out of date that it is actually not beneficial to have this online
2. It would make more sense to start from scratch using the markdown content in the `mdn/content` repository as the basis for reviving these languages.

With that in mind, we will be moving the content from the current [`mdn/translated-content`](https://github.com/mdn/translated-content) to the [`mdn/retired-archived-content`](https://github.com/mdn/retired-archived-content) repository and remove the options from the language drop-down on MDN Web Docs.

We appreciate your understanding in this matter. If you do want to reactivate these languages, please see the [process for activating a locale](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#activating-a-locale).

**Comment by SphinxKnight** [comment]
@schalkneethling is there an issue for this matter or should this "canonically" be tracked here?

(nitpicking: shouldn't `l10n-pl` be used as a label rather than `l10n-es?)

**Reply by schalkneethling** [reply]
Hey @SphinxKnight, not yet. I will open one this week. Thanks for the note on the labels, that was my mistake :)

**Comment by SphinxKnight** [comment]
Since https://github.com/mdn/translated-content/pull/8107 has been merged, this discussion can be closed/marked as answered
